typescript/web/src/components/import-button/import-images-modal/modal-url-list/url-list.tsx

```diff
@@ -19,9 +23,16 @@ export const UrlList = ({
 
   return (
     <VStack spacing={4} flex="1" alignItems="stretch">
-      <Button mt={2} onClick={() => setValue(imageSampleCollection.join("\n"))}>
-        Insert example images
-      </Button>
+      {isDevelopmentEnvironment && (
+        <Button
+          mt={2}
+          onClick={() => setValue(imageSampleCollection.join("\n"))}
+          background="pink.200"
+        >
+          <DevIcon size="1.5rem" mr="1" />
+          Insert example images
+        </Button>
```

Comment:
Nitpicking: 

I think a more standard way would be to do: 
```suggestion
        <Button
          mt={2}
          leftIcon={<DevIcon size="1.5rem" />}
          onClick={() => setValue(imageSampleCollection.join("\n"))}
          background="pink.200"
        >
          Insert example images
        </Button>
```

Reply:
Indeed 🤔 I tried this at first but it yielded very bad looking result so I went with the later. Guess the `size` part was off